doc/development/tutorials/theming-dev.rst

```diff
@@ -1,7 +1,5 @@
-.. highlight:: python
```

Comment:
As above, this isn't really a tutorial. Could we place it in `doc/development` rather than `doc/development/tutorials`? Also, drop the `-dev` suffix - it's obvious based on the folder it's in

Reply:
will do - I'm not sure what you mean about `-dev`...I don't see that any where in here?